### PR TITLE
Remove old dynatable

### PR DESCRIPTION
**Pull request by ehuelsmann**

### CODE REVIEW

File: lib/LedgerSMB/Scripts/payment.pm [hunk 1]
```diff
@@ -252,14 +278,16 @@ sub pre_bulk_post_report {
...
-        hiddens => $request,
-        columns => $cols,
-        heading => $heading,
-        rows    => $rows,
-        buttons => $buttons,
-    });
+    $request->{hiddens} = { %$request }; # prevent circular reference
+    delete $request->{form_id};
+    my $template = LedgerSMB::Template::UI->new_UI;
+    return $template->render($request, 'Reports/display_report',
```
Comment (ylavoie) [inline lib/LedgerSMB/Scripts/payment.pm]: render has 2 parameters.

File: lib/LedgerSMB/Scripts/setup.pm [hunk 2]
```diff
@@ -792,38 +792,43 @@ verify_check => md5_hex($check->test_query),
     my @skip_keys = grep /^skip_/, keys %$request;
     $hiddens->{@skip_keys} = $request->{@skip_keys};
 
+    my $cols = [];
+    for my $column (@{$check->columns // []}) {
```
Comment (ylavoie) [inline lib/LedgerSMB/Scripts/setup.pm]: display_cols are to be displayed. columns holds the editable values.

Comment (ylavoie) [reply lib/LedgerSMB/Scripts/setup.pm]: Also, shouldn't that be `for my $column (@{$check->columns} // []) {`

Comment (ylavoie) [reply lib/LedgerSMB/Scripts/setup.pm]: Nope, should not.

File: lib/LedgerSMB/Scripts/payment.pm (hunk 1 above)
Comment (ehuelsmann) [reply lib/LedgerSMB/Scripts/payment.pm]: Hmm. I thought it had 4: https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Template/UI.pm#L168 ?

File: lib/LedgerSMB/Scripts/setup.pm (hunk 2 above)
Comment (ehuelsmann) [reply lib/LedgerSMB/Scripts/setup.pm]: > display_cols are to be displayed. columns holds the editable values.
Working on a change which gets this right, right now.

**Comment by ehuelsmann** [comment]
This PR closes https://github.com/ledgersmb/LedgerSMB/projects/12 .

**Pull request merged**